### PR TITLE
chore: migrate to VitePress

### PR DESCRIPTION
**Pull request by Sepush**
i18n have some issue in detail so this PR still WIP.
- We need https://github.com/vuejs/vitepress/tree/feat/i18n be merged 
to fix https://github.com/vuejs/vitepress/issues/967
- Docs search after the PR be merge we need apply to docsearch
https://docsearch.algolia.com/apply/
then fill this todo under `/docs/.vitepress/config.ts` to enable it
image
- Some notes need to know can find in README file.

### CODE REVIEW

**Review by VoVAllen — APPROVED** [review]
LGTM! New doc site is on the way 🚀

**Review by kemingy — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -48,28 +45,35 @@ features:
 
 <script setup>
 import {
-  VPTeamPage,
-  VPTeamPageTitle,
-  VPTeamMembers
+  // VPTeamPage,
+  // VPTeamPageTitle,
+  // VPTeamMembers
```
Comment (Sepush) [inline docs/index.md:50]: these import are required

**Comment by aseaday** [comment]
Thanks this beautiful new version.
么么哒！

**Review by aseaday — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (gaocegege) [reply docs/index.md]: I am hiding the members page.

**Pull request merged**